### DISCUSSION TITLE
Vertical scrollbar always visible, how do I remove this?

### DISCUSSION THREAD

**Discussion by rhonautix** [rappasoft/laravel-livewire-tables#1141] [Q&A] [answered]
Hi!

I've noticed that there's a vertical scrollbar. How can this be removed? This is somewhat similar to [this](https://github.com/rappasoft/laravel-livewire-tables/issues/1045) but mine is a vertical scrollbar. 

No assets published, by the way.

![image](https://user-images.githubusercontent.com/115517521/230045971-ee7360f8-ef7e-47e0-90a0-1c7a7bed3361.png)

tailwind.config.js

![image](https://user-images.githubusercontent.com/115517521/230047372-68c2b0f4-3698-4907-9108-d3b5ced42861.png)

package.json

![image](https://user-images.githubusercontent.com/115517521/230048535-944e17f1-8a85-4fc3-a728-dbdeb188d67e.png)

**Comment by lrljoe** [comment, marked as answer]
The below should do it, this removes the default classes, and sets the below

```php
public function configure(): void
{
  $this->setTableWrapperAttributes([
    'default' => false,
    'class' => 'shadow border-b border-gray-200 dark:border-gray-700 sm:rounded-lg',
  ]);
}
```

Default is
"shadow overflow-y-scroll border-b border-gray-200 dark:border-gray-700 sm:rounded-lg"

**Comment by dmyers** [comment]
@lrljoe It might be good to set the default to just `overflow-y-auto`.

**Reply by lrljoe** [reply]
Not a bad call at all, please can you raise a feature request, or submit a PR for that, or I'll probably lose track of that!
